Cargo.toml

```diff
@@ -12,7 +13,15 @@ readme = "README.md"
 
 [dependencies]
 binrw = "0.13.3"
+crc16 = "0.4.0"
+heapless = "0.8.0"
+proptest = "1.6.0"
```

Comment:
dev-dependency?

Reply:
No you need these for the parser

Reply:
Oh I guess proptest can be a dev-dependency